x-pack/plugins/security_solution/public/flyout/shared/components/flyout_loading.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFlexItem, EuiLoadingSpinner } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { FLYOUT_LOADING_TEST_ID } from '../test_ids';
+
+/**
+ * Use this when you need to show a loading state in the flyout
+ */
+export const FlyoutLoading: React.VFC = () => (
+  <EuiFlexItem
```

Comment:
the only reason I put the spinner within a `EuiFlexItem` is because when it's being rendered in the DOM, it is actually right under a `EuiFlexGroup` coming from the expandable flyout framework (package)
![Screenshot 2023-09-13 at 9 38 31 AM](https://github.com/elastic/kibana/assets/17276605/bc41074e-0be0-4005-9df4-064946d6e60e)

Reply:
I just noticed that I did not do this for the new `FlyoutError` component, so just made that change there